### ISSUE TITLE
Feature request - Nagios perfdata

### ISSUE THREAD

**Issue by groupzer0** [PhaedrusTheGreek/nagioscheckbeat#12]
I am trying to figure out how to send Nagios 'perfdata' with the output type 'nagioscheck'. This is so that Nagios is adding the metric data to its perfdata file. I can't see a way to do this as the data types created by nagioscheckbeat separate the check status from the metric data into two types.

The data format which Nagios is looking for is outlined here, bu the essential format is:

status text followed by a pipe character (|), and then a string containing one or more performance data metrics

i.e.
PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80

Thank you!

**Comment by groupzer0** [comment]
I rewrote my old backend processes to pull the performance data from ES. It's a better process. 

so, I don't need this feature as of now. Maybe someone else could benefit from it.

**Comment by PhaedrusTheGreek** [comment]
OK glad to hear you have it working, I'll close this issue for now and if anyone else is interested , we can re-open.
